core/src/main/scala/monocle/std/NonEmptyList.scala

```diff
@@ -4,23 +4,23 @@ import monocle.function._
 import monocle.{PIso, Iso, Optional}
 
 import scalaz.NonEmptyList._
-import scalaz.{NonEmptyList, OneAnd}
+import scalaz.{IList, NonEmptyList, OneAnd}
 
 object nel extends NonEmptyListOptics
 
 trait NonEmptyListOptics {
 
   final def pNelToOneAnd[A, B]: PIso[NonEmptyList[A], NonEmptyList[B], OneAnd[List,A], OneAnd[List,B]] =
-    PIso((nel: NonEmptyList[A])    => OneAnd[List,A](nel.head, nel.tail))(
-      (oneAnd: OneAnd[List, B]) => NonEmptyList.nel(oneAnd.head, oneAnd.tail))
+    PIso((nel: NonEmptyList[A])    => OneAnd[List,A](nel.head, nel.tail.toList))(
+      (oneAnd: OneAnd[List, B]) => NonEmptyList.nel(oneAnd.head, IList.fromList(oneAnd.tail)))
 
```

Comment:
Maybe make an `Iso[NonEmptyList,IList]` and then compose with `Iso[IList,List]`?